mix.exs

```diff
@@ -36,56 +36,60 @@ defmodule Membrane.Gigachad.Mixfile do
 
   defp deps do
     [
-      {:membrane_core, "~> 0.11.3"},
-      {:membrane_h264_ffmpeg_plugin, "~> 0.25.4"},
-      {:membrane_file_plugin, "~> 0.13.2"},
-      {:membrane_common_c, "~> 0.14.0"},
-      {:membrane_mp4_plugin, "~> 0.19.0"},
-      {:membrane_tee_plugin, "~> 0.10.1"},
-      {:membrane_aac_plugin, "~> 0.13.0"},
-      {:membrane_rtp_h264_plugin, "~> 0.15.1"},
-      {:membrane_rtp_opus_plugin, "~> 0.7.0"},
-      {:membrane_rtp_plugin, "~> 0.21.0"},
-      {:membrane_funnel_plugin, "~> 0.7.0"},
-      # {:membrane_ice_plugin, "~> 0.14.3"},
```

Comment:
This has a problem with OpenSSL paths, so I think we shouldn't include any Webrtc stuff for now. We can either try to fix it or wait for Jellyfish to finish the work on ex_webrtc, where it should be solved.